platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -194,10 +194,7 @@ private void registerMbean() {
         }
       } catch (Exception e) {
         // objectName is not always non-null because new ObjectName(...) can throw an exception
-        LOGGER.info(
-            "Could not register MBean [{}].",
-            objectName != null ? objectName.toString() : KeystoreEditor.class.getName(),
-            e);
+        LOGGER.info("Could not register MBean [{}].", objectName.toString(), e);
```

Comment:
❗️  See above comment with same link. There's a better way to handle this finding along with the embedded `try` blocks. 
https://github.com/codice/ddf/blob/master/platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateGenerator.java#L63-L83